README.md

```diff
@@ -1,82 +1,103 @@
 [![pypi version](https://img.shields.io/pypi/v/spyder-vim.svg)](https://pypi.python.org/pypi/spyder-vim)
 [![Join the chat at https://gitter.im/spyder-ide/public](https://badges.gitter.im/spyder-ide/spyder.svg)](https://gitter.im/spyder-ide/public)
 [![OpenCollective Backers](https://opencollective.com/spyder/backers/badge.svg?color=blue)](#backers)
 [![OpenCollective Sponsors](https://opencollective.com/spyder/sponsors/badge.svg?color=blue)](#sponsors)
 
 ## Build status
+
 ![Linux tests](https://github.com/spyder-ide/spyder-vim/workflows/Linux%20tests/badge.svg)
 ![Macos tests](https://github.com/spyder-ide/spyder-vim/workflows/Macos%20tests/badge.svg)
 ![Window tests](https://github.com/spyder-ide/spyder-vim/workflows/Windows%20tests/badge.svg)
 [![Coverage Status](https://coveralls.io/repos/github/spyder-ide/spyder-vim/badge.svg?branch=master)](https://coveralls.io/github/spyder-ide/spyder-vim?branch=master)
 
 ----
 
-## Important Announcement: Spyder is unfunded!
-
-Since mid November/2017, [Anaconda, Inc](https://www.anaconda.com/) has
-stopped funding Spyder development, after doing it for the past 18
-months. Because of that, development will focus from now on maintaining
-Spyder 3 at a much slower pace than before.
-
-If you want to contribute to maintain Spyder, please consider donating at
-
-https://opencollective.com/spyder
+# Overview
 
-We appreciate all the help you can provide us and can't thank you enough for
-supporting the work of Spyder devs and Spyder development.
+Spyder Plugin for executing Vim commands inside the code editor. Although it does not currently support many VIM commands, it will be developed gradually.
 
-If you want to know more about this, please read this
-[page](https://github.com/spyder-ide/spyder/wiki/Anaconda-stopped-funding-Spyder).
+<img src="/doc/example.gif" width="600">
 
-----
+| Category     | supported commands                                                            |
+| ------------ | ----------------------------------------------------------------------------- |
+| Movement     | h, j, k, l, w, b, e, space, backspace, return, $, 0, ^, G, gg, zz, H, L, M, % |
+| Change       | x, r, o, O, u, d, dd, dw, D, c, cc, cw, J, ~, <, >, <<, >>                    |
+| Copy & Paste | yy, yw, y$, p, P                                                              |
+| Search       | /, ?, n, N, f, F                                                              |
+| mode         | i, I, a, A, v, V                                                              |
+| Register     | -, 0, 1, unamed                                                               |
+| File         | ZZ, gt, gT, :w, :q, :wq, :n, :e                                               |
 
 ## Installation
+
 To install this plugin, you can use either ``pip`` or ``conda`` package managers, as it follows:
 
 Using pip:
+
 ```
 pip install spyder-vim
 ```
 
 Using conda:
+
 ```
 conda install spyder-vim -c spyder-ide
```

Comment:
Lets change here the conda channel to be `conda-forge`. So this will become something like:

```suggestion
conda install spyder-vim -c conda-forge
```